src/controllers/albumController.js

```diff
@@ -7,11 +7,35 @@ const createAlbum = async (req, res) => {
   req.body.user = req.user.userId;
   const album = await Album.create(req.body);
   res.status(StatusCodes.CREATED).json({ album });
+  /*
+     #swagger.summary = 'Create new album and save it to the database'
+            #swagger.parameters['newAlbum'] = {
+        in: 'body',
+        description: 'Album information to use for creation.',
+        required: true,
+        type: 'object',
+        schema: { $ref: '#/definitions/NewAlbum' }
+     } 
+     #swagger.responses[201] = {
+				description: 'Albums was successfully created.',
+        schema: [{ $ref: '#/definitions/Album' }]
```

Comment:
The create endpoint returns a single album, rather than a list that this line is documenting and it will be wrapped in an object with the key `"album"`

![](https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/9094098/933298ac-58ab-41fa-88c3-f53e8353d0c3)

so this will need to be:

```suggestion
        schema: { album: { $ref: '#/definitions/Album' } }
```

Reply:
@akosasante Akos, I am wondering where do we get this open.spotify.com... url because when we fetch the data in mongo we save https://api.spotify.com/v1/albums/6r1lh7fHMB499vGKtIyJLy
![Captura de pantalla 2023-09-01 a las 0 58 29](https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/103755090/eae8a2eb-4ac8-43b7-9f8c-b8d4200d1a10)
 the same url we get from api docs https://developer.spotify.com/documentation/web-api/reference/get-an-album

ans so i am wonderying how this url passes the validator in Album.js?   ```
spotifyUrl: {
      type: String,
      required: true,
      validate: {
        validator: function (v) {
          return /^https:\/\/open\.spotify\.com\/album\/[a-zA-Z0-9]+$/.test(v); //ensures that the URL starts with https://open.spotify.com/album/ followed by alphanumeric characters.
        },
        message: 'Invalid Spotify URL',
      },
```

Reply:
// for mi -strart commiting / updating from here